osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/Components/PathControlPointVisualiser.cs

```diff
@@ -402,7 +403,14 @@ private MenuItem createMenuItemForPathType(PathType? type)
             int totalCount = Pieces.Count(p => p.IsSelected.Value);
             int countOfState = Pieces.Where(p => p.IsSelected.Value).Count(p => p.ControlPoint.Type == type);
 
-            var item = new TernaryStateRadioMenuItem(type == null ? "Inherit" : type.ToString().Humanize(), MenuItemType.Standard, _ =>
+            string name = type switch
+            {
+                null => "Inherit",
+                PathType.BSpline => "B-Spline",
+                _ => type.ToString().Humanize()
+            };
```

Comment:
Rather than do this I'd much prefer to stop relying on humanizer at all and instead explicitly annotate the enum members with `[Description]` or `[LocalisableDescription]`.